xapian-core/backends/honey/honey_version.h

```diff
@@ -276,7 +295,9 @@ class HoneyVersion {
 		     Xapian::termcount o_doclen_ubound,
 		     Xapian::termcount o_wdf_ubound,
 		     Xapian::totallength o_total_doclen,
-		     Xapian::termcount o_spelling_wordfreq_ubound);
+		     Xapian::termcount o_spelling_wordfreq_ubound,
+		     Xapian::termcount o_uniq_terms_lbound = 0,
```

Comment:
@olly, my comment in trac was about this change. If default values are removed, we get a compile error.

Reply:
So the problem is when this gets called from `honey_compact.cc` when converting a database from glass?

So in that case, we're setting this to zero, but then working out the correct bounds as we process the termlist.  So basically this is OK, but think it's probably better to explicitly pass `0` from the caller instead of having a default which might get inadvertently used in other situations where it isn't appropriate.

Reply:
Yes. That was the problem. I will do that